packages/kokoas-client/src/pages/projSearch/sections/result/details/paymentDetails/unissuedInvoiceAlert/helper/getDisplayPaymentDate.ts

```diff
@@ -0,0 +1,10 @@
+import format from 'date-fns/format';
+import isValid from 'date-fns/isValid';
+
+
+export const getDisplayPaymentDate = (paymentDate: Date | null) => {
+  
+  if (!paymentDate) return '';
+  return isValid(paymentDate) ? format(new Date(paymentDate), 'yyyy年M月d日') : '';
```

Comment:
isValidで合格すると、生のDate型になり、datefns.format()で行けるかな。

```suggestion
  return isValid(paymentDate) ? format(paymentDate, 'yyyy年M月d日') : '';
```

Reply:
ご指摘ありがとうございます。
動作テストも問題なかったため、上記の通り修正しました。
[a2771e1](https://github.com/cocosumo/yumecoco-monorepo/pull/1250/commits/a2771e15e2d88e109ccee897128ca031ee3a1ff0)にて、ご確認をお願いいたします。